### ISSUE TITLE
Stuck on "loading language traineddata"

### ISSUE THREAD

**Issue by chukwumaijem** [naptha/tesseract.js#439]
**Describe the bug**
I have a simple html with script tags as follows.

```
  <script src='https://unpkg.com/tesseract.js@v2.1.0/dist/tesseract.min.js'></script>
  <script>
    window.onload = async () => {
      const worker = Tesseract.createWorker({
        logger: m => console.log(m)
      });

      await worker.load();
      await worker.loadLanguage('eng');
      await worker.initialize('eng');
      const recData = await worker.recognize('https://tesseract.projectnaptha.com/img/eng_bw.png');

      console.log('=====recData===', recData);
      await worker.terminate();
    }
  </script>
```
When the page loads, i the following logs on the console, and nothing else happens. What am I doing wrong?
<img width="834" alt="Screenshot 2020-04-14 at 07 49 48" src="https://user-images.githubusercontent.com/17248267/79194485-88f68080-7e24-11ea-80bd-7652dc376c08.png">

**Environment**
 - OS: MacOS Catalina
 - Browser [e.g. chrome version 80]

**Comment by jeromewu** [comment]
Most of the time it is a network issue.

You can try to use the offline version first and see if it works: https://github.com/jeromewu/tesseract.js-offline

**Comment by iamkoks** [comment]
Faced the same problem, after connecting this  <script src='https://unpkg.com/tesseract.js@v2.1.0/dist/tesseract.min.js'></script>, it worked

**Comment by nrmhrk** [comment]
adding `cachemethod: 'none'` in worker constructor solved this issue for me. 
loading data and workers from CDN is really the worst design choice in this wrapper.

**Comment by Balearica** [comment]
Closed as duplicate of #414.  I will look into why the promise returned by `loadLanguage` never resolves, and post findings on that issue.
